Exercise_1.java

```diff
@@ -3,6 +3,20 @@
     int binarySearch(int arr[], int l, int r, int x) 
     { 
         //Write your code here
+        if(l <= r) {
+        
+            int mid = (l + r) / 2;

```

Comment:
try to avoid integer overflow while calculating mid

Reply:
sure, thank you for the feedback.